pkcs12.go

```diff
@@ -66,12 +66,13 @@ type encryptedPrivateKeyInfo struct {
 func (i encryptedPrivateKeyInfo) GetAlgorithm() pkix.AlgorithmIdentifier { return i.AlgorithmIdentifier }
 func (i encryptedPrivateKeyInfo) GetData() []byte                        { return i.EncryptedData }
 
-// Error indicating that the input is in an unsupported format. The format may be allowed by the RFC's, but may not be implemented by this package.
+// UnsupportedFormat Error indicates that the input is in an unsupported format.
+// The format may be allowed by the RFC's, but may not be implemented by this package.
 type UnsupportedFormat string
```

Comment:
If they are just logging the error, true. But if checking which error with a type assertion, `err.(pkcs12.UnsupportedFormat)` they will get a compile time error.